utils/vectorscan/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/VectorCamp/vectorscan/tar.gz/$(PKG_NAME)/$(PKG_VERSION)?
+PKG_HASH:=e61c78f26a9d04ccffab0df1159885c4503fc501172402c57f7357a2126ea3c6
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=BSD
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_PARALLEL:=1
+
+HOST_BUILD_DEPENDS:=ragel/host ninja/host python/host boost/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/vectorscan
+  CATEGORY:=Utilities
+  SECTION:=utils
+  TITLE:=Vectorscan
+  URL:=https://github.com/VectorCamp/vectorscan
+  MENU:=1
+endef
+
+define Package/vectorscan/description
+  A fork of Intel's Hyperscan, modified to run on more platforms. Currently ARM
+  NEON/ASIMD is 100% functional, and Power VSX are in development. ARM SVE2
+  will be implemented when harwdare becomes accessible to the developers.
+  More platforms will follow in the future, on demand/request.
+endef
+
+CMAKE_OPTIONS += \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=lib \
+	-DBUILD_SHARED_LIBS=ON \
+	-Wno-dev
+
+define Package/vectorscan/install
+	$(INSTALL_DIR) $(1)/usr/include/hs
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/usr/include/hs/hs.h $(1)/usr/include/hs/hs.h
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/usr/include/hs/hs_common.h $(1)/usr/include/hs/hs_common.h
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/usr/include/hs/hs_compile.h $(1)/usr/include/hs/hs_compile.h
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/usr/include/hs/hs_runtime.h $(1)/usr/include/hs/hs_runtime.h
```

Comment:
Since the build from upstream is currently broken, I cannot verify your statement about the need for them.  I can tell you that other distros package these together and that snort uses hyperscan successfully with them on the file system.

Reply:
I added a `Build/InstallDev` per your request.